src/ResourceManager/Network/Commands.Network.Test/Commands.Network.Test.csproj

```diff
@@ -352,6 +359,10 @@
     <None Include="SessionRecords\Commands.Network.Test.ScenarioTests.NetworkCloudExceptionTests\TestNotFound.json">
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </None>
+    <None Include="SessionRecords\Commands.Network.Test.ScenarioTests.NetworkProfileTests\TestContainerNetworkInterfaceConfigCRUD.json" />
```

Comment:
@jdhuntington  the tests failed most likely because the entries are missing :
```
      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory> 
    </None> 
```

the entry should be :
```
    <None Include="SessionRecords\Commands.Network.Test.ScenarioTests.NetworkProfileTests\TestContainerNetworkInterfaceConfigCRUD.json" > 
      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory> 
    </None> 
```
do the same for all the four json files

Reply:
fixed in github client - let's see if it passes now.